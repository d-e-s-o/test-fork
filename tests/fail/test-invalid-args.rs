// Copyright (C) 2025 Daniel Mueller <deso@posteo.net>
// SPDX-License-Identifier: (Apache-2.0 OR MIT)

/// Unsupported inner attribute.
#[test_fork::test(inner_test)]
fn invalid_args() {}

fn main() {}
