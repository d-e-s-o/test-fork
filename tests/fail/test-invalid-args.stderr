error: test_fork::test does not currently accept arguments
 --> tests/fail/test-invalid-args.rs:5:19
  |
5 | #[test_fork::test(inner_test)]
  |                   ^^^^^^^^^^
